Add how to select image

AseiSugiyama opened the pull request:
1. Add `how_to_select.md` describes two docker image contained in this repo. 
2. Add link to [jupyter/docker-stacks](https://github.com/jupyter/docker-stacks) in `README.me`

AseiSugiyama committed:
add how_to_select.md

AseiSugiyama committed:
add link to jupyter/docker-stacks in README.md

mrkn commented on README.md:
```diff
@@ -3,13 +3,16 @@
 
 https://mybinder.org/v2/gh/RubyData/docker-stacks/master?filepath=README.ipynb
 
+If you [have Docker installed](https://docs.docker.com/install/) know [which Docker image](how_to_select.md) you want to use, and want to launch a single Jupyter Notebook server in a container.
```
This sentence seems not correct English (or my English skill is not enough to recognize it).
Could you please change this sentence to the following simpler English?

```
Otherwise, you can launch a single Jupyter Notebook server in a Docker container.  Two Docker images are available. You can know which is suitable for you in [this document](how_to_select.md).
```

AseiSugiyama replied:
Thank you for your suggestion. I applied it.

mrkn commented on how_to_select.md:
```diff
@@ -0,0 +1,36 @@
+# Selecting an Image
+
+We have two Docker images followings:
```
Could you please remove `followings`?

AseiSugiyama replied:
Done.

mrkn commented on how_to_select.md:
```diff
@@ -0,0 +1,36 @@
+
+## Ruby
+
+These two images contains same Ruby components for data science followings:
```
Could you modify this by the following?

```
These two images contain the same Ruby components for data science:
```

AseiSugiyama replied:
Done.

mrkn commented on how_to_select.md:
```diff
@@ -0,0 +1,36 @@
+
+## `rubydata/minimal-notebook`
+
+`rubydata/minimal-notebook` consists of minimal components for data science using Ruby, Python.
```
Please replace `Ruby, Python.` by `Ruby and Python.`

AseiSugiyama replied:
Done.

mrkn commented:
@AseiSugiyama Thank you for your pull-request.

I leave some comments.  Could you please fix for them?

AseiSugiyama commented:
@mrkn Sure, let me try.

AseiSugiyama committed:
fix incorrect sentence of README.md

https://github.com/RubyData/docker-stacks/pull/19#discussion_r284078525

AseiSugiyama committed:
fix unfluent description of container components of Ruby

https://github.com/RubyData/docker-stacks/pull/19#discussion_r284078845

AseiSugiyama committed:
fix influent description of Docker image in how_to_select.md

https://github.com/RubyData/docker-stacks/pull/19#discussion_r284078644
https://github.com/RubyData/docker-stacks/pull/19#discussion_r284078938

AseiSugiyama commented:
@mrkn 
I've finished fixing issues. Could you check it again?

mrkn commented:
@AseiSugiyama thank you very much!!
